src/resources/js/directives.js

```diff
@@ -1971,7 +1971,7 @@
             },
             template: function() {
                 return  '<div class="form-group form-side-by-side" ng-class="{\'input--hide-label\': i18n}">' +
-                            '<div class="form-side form-side-label">' +
+                            '<div class="form-control no-border">' +
```

Comment:
the bs4 class `border-0` would do the trick.
see: https://getbootstrap.com/docs/4.0/utilities/borders/

Reply:
thanks